### PR TITLE
Created nitesh.json

### PR DESCRIPTION
**Pull request by Moon1Knight**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by Moon1Knight** [comment]
Created nitesh.json #10304

File: data/nitesh.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "FaLink",
+      "url": "https://anushkaraj.me"
+    }
+  ],
+  "links": [
+    {
+      "name": "Follow my Open Source on GitHub",
+      "url": "https://github.com/Moon1knight",
+      "icon": "FaGithub"
+    },
```
Comment (kumarsonsoff3) [inline data/nitesh.json:21]: Please remove this trailing comma here! 

```suggestion
    }
```

**Review by kumarsonsoff3 — APPROVED** [review]
Filename must be exactly same as your GitHub username. Please rename your [filename here](https://github.com/Moon1Knight/BioDrop/edit/main/data/nitesh.json) to `Moon1Knight.json`

Change `nitesh.json`  -->  `Moon1Knight.json`

Also make the below suggested change.

**Comment by eddiejaoude** [comment]
I will close as the PR has been open for a long time with no changes from the author

**Pull request closed without merging**